README.rst

```diff
@@ -0,0 +1,64 @@
+
+==================================
+Watchdog Formula
+==================================
+
+Install and configure watchdog daemon
+
+
+Sample Pillars
+==============
+
+Single watchdog service
+
+.. code-block:: yaml
+
+    watchdog:
+      server:
+        enabled: true
+        timeout: 60
+        # Salt Stack will automatically detect the necessary kernel module which needs to be loaded (ex. hpwdt). You may specify the kernel parameters if needed:
+        kernel:
+          parameter:
+            soft_panic: 1
+            parameter: value
```

Comment:
remove this

Reply:
Why? It describe how to use this module... I prefer to have some examples there, because then users know how to use it...

Reply:
> We usually don't using examples which haven't any sense. So please use only relevant keys and values (but in this case it can be accepted this way).